handlers/handler-flowcontrol-qps/src/main/java/org/apache/servicecomb/qps/QpsControllerManager.java

```diff
@@ -200,13 +201,18 @@ private void createQpsControllerIfNotExist(String configKey) {
   }
 
   protected void updateObjMap(String configKey) {
-    for (Entry<String, AbstractQpsStrategy> controllerEntry : qualifiedNameControllerMap
-        .entrySet()) {
-      if (keyMatch(configKey, controllerEntry)) {
-        AbstractQpsStrategy qpsStrategy = searchQpsController(controllerEntry.getKey());
-        controllerEntry.setValue(qpsStrategy);
-        LOGGER.info("QpsController updated, operationId = [{}], configKey = [{}], qpsLimit = [{}]",
-            controllerEntry.getKey(), qpsStrategy.getKey(), qpsStrategy.getQpsLimit());
+    Iterator<Entry<String, AbstractQpsStrategy>> it = qualifiedNameControllerMap.entrySet().iterator();
```

Comment:
Can you add a test case for this issue? see QpsControllerManagerTest for examples